### DISCUSSION TITLE
Switch build system to `Makefile` instead of scripts?

### DISCUSSION THREAD

**Discussion by daPhipz** [Schneegans/Fly-Pie#128] [Ideas & Feature Discussions]
I recently dug into learning `make`, and now want to apply the things that I learned in the "real world".
AFAIK, the building of Fly-Pie happens through the `scripts/create-release.sh` script as of now - right?
While this eventually will build the zip, `make` has the advantage of being able to only re-build the things that have changed since the last build (and therefore taking considerably less time to finish) - but I'm sure you know that already :wink:

Are you open to changing the build system to `make`? As I said, I would happily do the work of porting the existing one since I want to practice a little bit ;)

**Comment by Schneegans** [comment]
Hey! Nice to see you again :)

I would say: Just give it a go! Build performance is not really an issue (the complete compile-zip-install process takes 300 ms on my machine), but people are actually quite used to to autotools so newcomers may like it. And you could basically merge all the scripts into one makefile.

**Reply by daPhipz** [reply]
Yeah, I've been silent for quite some time - but I always had an eye on the developments :)
I'll transfer this discussion into an Issue, since you are interested in giving it a go!
